### PR TITLE
Kubearmor GSoC 2024

### PR DESCRIPTION
**Pull request by Shreyas220**

### CODE REVIEW

**Comment by aliok** [comment]
@Shreyas220 thanks for submitting the proposal.

Can we get a 👍 from the [project maintainers](https://raw.githubusercontent.com/cncf/foundation/main/project-maintainers.csv) like @daemon1024 ?

Thanks @daemon1024 @primalpimmy @DelusionalOptimist and @Shreyas220!

**Comment by aliok** [comment]
Please note that GSoC is a program known for its strict deadlines. In addition to responding to your mentee on time, you will be required to submit evaluations on time. Failures to meet the deadlines might affect CNCF's future participation in GSoC.

**Review by daemon1024 — APPROVED** [review]
Thanks @Shreyas220, LGTM. We are looking forward to GSoC 2024 with CNCF @aliok

**Comment by aliok** [comment]
@nate-double-u leaving the merge to you.

File: programs/summerofcode/2024.md [hunk 1]
```diff
@@ -111,6 +111,28 @@ You can find the project ideas from previous year [here](./2023.md).
...
+- Mentor(s):
+  - Barun Acharya (@daemon1024, barun1024@gmail.com) - primary
+  - Shreyas Mishra (@Shreyas220, shreyas.ny@gmail.com)
+  - Prashant Mishra (@primalpimmy, prashant20.pm@gmail.com)
+  - Rudraksh Pareek (@DelusionalOptimist, rudrakshpareek3601@gmail.com)
+
+- Upstream Issue: kubearmor/KubeArmor#1669
+
+
+
```
Comment (nate-double-u) [inline programs/summerofcode/2024.md:135]: Removing extra line breaks.
```suggestion
```

**Review by nate-double-u — APPROVED** [review]
Thanks everyone!

**Pull request merged**